lib/rubocop/cop/rails/compact_blank.rb

```diff
@@ -120,7 +122,11 @@ def offense_range(node)
         end
 
         def preferred_method(node)
-          node.method?(:reject) || node.method?(:select) ? 'compact_blank' : 'compact_blank!'
+          if %i[reject select filter].include?(node.method_name)
+            'compact_blank'
+          else
+            'compact_blank!'
+          end
```

Comment:
Can you make this a constant?

```suggestion
          DESTRUCTIVE_METHODS.include?(node.method_name) ? 'compact_blank!' : 'compact_blank'
```

Reply:
Thank you for your feedback. I've fixed it!